Neos.Neos/Classes/Command/SiteCommandController.php

```diff
@@ -220,4 +327,51 @@ function (Site $site) use ($siteNodePattern) {
             }
         );
     }
+
+    protected function determineTargetPath(?string $packageKey, ?string $path): string
+    {
+        $exceedingArguments = $this->request->getExceedingArguments();
+        if (isset($exceedingArguments[0]) && $packageKey === null && $path === null) {
+            if (file_exists($exceedingArguments[0])) {
+                $path = $exceedingArguments[0];
+            } elseif ($this->packageManager->isPackageAvailable($exceedingArguments[0])) {
+                $packageKey = $exceedingArguments[0];
+            }
+        }
+        if ($packageKey === null && $path === null) {
+            $this->outputLine('<error>You have to specify either <em>--package-key</em> or <em>--path</em></error>');
+            $this->quit(1);
+        }
+        if ($path === null) {
+            $package = $this->packageManager->getPackage($packageKey);
+            $path = Files::concatenatePaths([$package->getPackagePath(), 'Resources/Private/Content']);
+        }
+        if (str_starts_with($path, 'resource://')) {
+            $this->outputLine('<error>Resource paths are not allowed, please use <em>--package-key</em> instead or a real path.</error>');
+            $this->quit(1);
+        }
```

Comment:
todo are we okay with ignoring this for now, or should we transform the `resource` path to a real path? https://github.com/neos/neos-development-collection/issues/4912

(https://github.com/neos/flow-development-collection/issues/2687)